src/main/java/it/uniroma3/mat/extendedset/intset/ConciseSet.java

```diff
@@ -390,7 +390,7 @@ private static int getFlippedBit(int word)
    */
   private static int getLiteralBitCount(int word)
   {
-    return BitCount.count(getLiteralBits(word));
+    return Integer.bitCount(getLiteralBits(word));

```

Comment:
Is Integer.bitCount faster than BitCount.count? How much? Under what conditions?